test/cpp/end2end/end2end_test.cc

```diff
@@ -2145,8 +2147,14 @@ TEST_P(SecureEnd2endTest, ClientAuthContext) {
   std::vector<grpc::string_ref> tst =
       auth_ctx->FindPropertyValues("transport_security_type");
   ASSERT_EQ(1u, tst.size());
-  EXPECT_EQ(GetParam().credentials_type, ToString(tst[0]));
-  if (GetParam().credentials_type == kTlsCredentialsType) {
+  std::string credentials_type = GetParam().credentials_type;
+  if (GetParam().credentials_type == kTls12CredentialsType ||
+      GetParam().credentials_type == kTls13CredentialsType) {
+    credentials_type = kTlsCredentialsType;
+  }
+  EXPECT_EQ(credentials_type, ToString(tst[0]));
```

Comment:
Since we're really checking that `ToString(tst[0])` is the correct transport security type, could we rename `credentials_type` to `transport_security_type` instead?